CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# librdkafka v1.9.4
+
+librdkafka v1.9.4 is a maintenance release:
+
+* Added rd_kafka_sasl_set_credentials API function to set or reset the SASL PLAIN or SCRAM credentials used for new broker connections.
```

Comment:
This should go in the v1.9.3 section below - that's the next release.
Add it to the Enhancement chapter
Try to stick below 80 columns

Reply:
Fixed